lib/sqlalchemy/sql/selectable.py

```diff
@@ -5196,7 +5231,7 @@ def filter_by(self: SelfSelect, **kwargs: Any) -> SelfSelect:
         from_entity = self._filter_by_zero()
 
         clauses = [
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325